tests/end-to-end/external-solution-integration/pkg/testsuite/create_mapping.go

```diff
@@ -3,24 +3,23 @@ package testsuite
 import (
 	"fmt"
 
-	acApi "github.com/kyma-project/kyma/components/application-broker/pkg/apis/applicationconnector/v1alpha1"
-	acClient "github.com/kyma-project/kyma/components/application-broker/pkg/client/clientset/versioned/typed/applicationconnector/v1alpha1"
+	acv1alpha1 "github.com/kyma-project/kyma/components/application-broker/pkg/apis/applicationconnector/v1alpha1"
+	acclientset "github.com/kyma-project/kyma/components/application-broker/pkg/client/clientset/versioned/typed/applicationconnector/v1alpha1"
```

Comment:
Previously you used `appbrokerclientset`, let's be consistent.

Reply:
i lost track 

we need a tool!

Reply:
Totally :(